features/authentication/components/account-switcher.tsx

```
@@ -0,0 +1,99 @@
+    },
+    [navigation],
+  )
+
+  return (
+    <HStack style={themed($titleContainer)}>
+      {!noAvatar && <ProfileAvatar />}
+      <DropdownMenu
+        style={themed($dropDownMenu)}
+        onPress={onDropdownPress}
+        actions={[
+          ...preferredSendersDisplayInfos?.filter(Boolean)?.map((profile) => ({
```

Biome (1.9.4)

[error] Unsafe usage of optional chaining.

If it short-circuits with 'undefined' the evaluation will throw TypeError here:


(lint/correctness/noUnsafeOptionalChaining)